Discovery/Web-Content/hashicorp-consul-api.txt

```diff
@@ -0,0 +1,41 @@
+# HashiCorp Consul RESTful API Port 8500
+
```

Comment:
I think these first two lines should be removed, since they don't apport anything to the list, and they're the only comment in the entire list. If you feel more context is needed for this list, you should add it to a README file that's in the same folder

Reply:
Removed the two lines